locales/nl/nl.json

```diff
@@ -3,8 +3,8 @@
     "60 Days": "60 dagen",
     "90 Days": "90 dagen",
     ":amount Total": ":amount Totaal",
-    ":resource Details": ":resource Details",
-    ":resource Details: :title": ":resource Details: :title",
+    ":resource Details": ":resource Informatie",
```

Comment:
Google does not translate the word "Details" into Dutch. Therefore, I took as a basis the translated word "Details" from [file](https://github.com/Laravel-Lang/lang/pull/1644/files#diff-956387d70e3715ae6821373868ebb510d93dfc1570169cad7e09e9f412aeb62dL163).

@WhereIsLucas, can you tell i did the right thing?

![image](https://user-images.githubusercontent.com/10347617/115685068-919b6400-a360-11eb-8cbb-c97bd10eff9f.png)